Tests/Admin/Extension/LockExtensionTest.php

```diff
@@ -66,35 +66,19 @@ public function testConfigureFormFields()
     {
         $formMapper = $this->configureFormMapper();
         $form = $this->configureForm();
-        $event = new FormEvent($form, array());
 
-        $this->modelManager->expects($this->once())
-            ->method('getLockVersion')
-            ->will($this->returnValue(1));
+        $this->configureAdmin(null, null, $this->modelManager->reveal());
```

Comment:
@verheyenkoen this is where it comes from: a merge. Learn interactive rebase, get rid of this commit and you should be good.